api/v1alpha6/conversion_test.go

```diff
@@ -102,6 +102,15 @@ func TestFuzzyConversion(t *testing.T) {
 				}
 			},
 
+			func(status *infrav1.OpenStackClusterStatus, c fuzz.Continue) {
+				c.FuzzNoCustom(status)
+
+				// this field does not exist in v1alpha6
+				if status.APIServerLoadBalancer != nil {
+					status.APIServerLoadBalancer.LoadBalancerNetwork = nil
+				}
+			},
```

Comment:
This sounds wrong, we should probably add restore functions instead of blocking such fuzzing.

Reply:
Yes. There are several examples in v1alpha6 and v1alpha7.

Reply:
thanks will take a closer look..